File: fortran/fortran77/Go/fortran77_lexer_base.go
package parser

import (
    "github.com/antlr4-go/antlr/v4"
)

type Fortran77LexerBase struct {
    *antlr.BaseLexer
    lastToken antlr.Token
}

func (l *Fortran77LexerBase) IsColumnZero() bool {
    return l.GetCharPositionInLine() == 0;
}

File: fortran/fortran77/TypeScript/Fortran77LexerBase.ts
import {Lexer, Token, CharStream} from "antlr4";

export default class Fortran77LexerBase extends Lexer {
    constructor(input: CharStream) {
        super(input);
    }

    IsColumnZero(): boolean {
	return this.column == 0;
    }
}
